.github/workflows/python.yaml

```diff
@@ -0,0 +1,24 @@
+name: Python
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+    tags:
+      - "*"
+  pull_request:
+    paths:
+      - "python/**"
```

Comment:
also needs requirements.txt and tox.ini in the root.

and ./scripts/ci_prepare_python.bash (and similar for the other helper scripts for other langs)